flink-ml-lib/src/test/java/org/apache/flink/ml/feature/ImputerTest.java

```diff
@@ -265,9 +265,7 @@ public void testNoValidDataOnMedianStrategy() {
         final List<Row> trainData =
                 new ArrayList<>(
                         Arrays.asList(
-                                Row.of(Double.NaN, Float.NaN),
-                                Row.of(null, null),
-                                Row.of(1.0, 1.0f)));
+                                Row.of(Double.NaN, 3.0f), Row.of(null, 2.0f), Row.of(1.0, 1.0f)));
```

Comment:
Why do we need to change the value here as part of this PR?

Reply:
Before the change, the 2 input columns both consisted of missing data, so the both 2 columns are invalid. And the aggregation summaries of columns are stored in a HashMap, which is not order-preserving, so we are not sure which column is detected first.